### PR TITLE
Chores feature

### PR DESCRIPTION
**Pull request by webshuriken**
I have implemented the Chores pages with the following features:

- ability to create many chore lists
- remove the chore list with a confirmation modal
- add items into each list
- each item can be marked as completed
- completed items are labelled with the user name of the person that marked them as complete
- remove individual chore items, with a confirmation modal

### CODE REVIEW

File: prisma/schema.prisma
[the remark this answers is not in the document]
Comment (webshuriken) [reply prisma/schema.prisma:1]: This is actually something i wanted to discuss with you on wednesday.. i think that makes sense but when we get the chores item data will it also retrieve the user data for the linked user or do we need to makr separate request to get the infor for thebusrr for the chor3s item??

[the remark this answers is not in the document]
Comment (AliMcG) [reply prisma/schema.prisma:1]: I think we will need the user data for the field "completedBy" - we will need the name and id of the connected user and we might even want to have more data to preform more checks - i.e. to confirm that the User has permission to interact with the list.

[the remark this answers is not in the document]
Comment (webshuriken) [reply prisma/schema.prisma:1]: Sp instead of linking the user to the User table we just store their name and id as part of the competedBy field?

**Review by webshuriken — COMMENTED** [review]
Nice.

**Pull request merged**